### DISCUSSION TITLE
Can't get easy effects to show input and output devices

### DISCUSSION THREAD

**Discussion by Josh65-2201** [wwmm/easyeffects#2787] [Q&A]
I'm trying to get VLC audio to play out with no modifications and mic input to be modified then go out. I have a USB sound card and have installed pipewire, pipewire-pulse, pipewire-alsa. Installing pipewire brakes VLC audio output and easyeffects show no devices in the input output pipewire section

**Comment by wwmm** [comment]
>Installing pipewire brakes VLC audio output and easyeffects show no devices in the input output pipewire section

Something definitely went wrong while installing PipeWire. Which distribution are you using? The ones that did not move to PipeWire by default yet usually have a tutorial somewhere explaining how to remove Pulseaudio and installing PipeWire.

**Reply by Josh65-2201** [reply]
I'm using Dietpi 8.23.3. I have use tutorials on installing pipewire but they give the same broken result.

**Comment by Josh65-2201** [comment]
So I have reset my install. using Diet PI 9.2 I have VLC working and playing audio and easy effects is install with pipewire, but with the same issue of showing no inputs or outputs.

**Reply by wwmm** [reply]
Run it in debug mode as explained at https://github.com/wwmm/easyeffects/wiki/Reporting-Bugs#basic-logs so we can see if there are errors preset in its logs.

**Reply by Josh65-2201** [reply]
It doesn't open anymore, This is all it giving me
![image](https://github.com/wwmm/easyeffects/assets/26977075/d34d4724-90de-4ad6-ada5-2222d334cbb6)

**Reply by wwmm** [reply]
Did you kill its process or forced it to exit `easyeffects -q` before trying to get the logs? There should definitely be more output.

**Reply by Josh65-2201** [reply]
Forgot about that, here is the output
[log.txt](https://github.com/wwmm/easyeffects/files/15170883/log.txt)

**Reply by wwmm** [reply]
>pipe_manager.cpp:1311	found metadata: settings

There should be a default metadata too. Usually when it is missing the reason is not having installed the package `pipewire-pulse`. Although EasyEffects does not need the pulseaudio compatibility layer PipeWire is pacakge by distributions in a way that the code providing the default metadata is in the pipewire-pulse package.

**Reply by Josh65-2201** [reply]
I install pipewire-pluse and rebooted and they still don't show anything, here's the log
[log.txt](https://github.com/wwmm/easyeffects/files/15171188/log.txt)

**Reply by wwmm** [reply]
Which PipeWire version did you install? EasyEffects is detecting `0.3.65` that is too old and will probably not work with EasyEffects `7.0.0`. And even EasyEffects version is too old and does not have many of the fixes available in `7.1.6`.

**Reply by Josh65-2201** [reply]
I used the `sudo apt <package name>` and what ever they download by default. Since they aren't on the Dietpi software downloader.

**Reply by wwmm** [reply]
Something you could try is installing EasyEffects through a flatpak package https://github.com/wwmm/easyeffects/wiki/Package-Repositories#flatpak. But with a PipeWire pacakge this old the chances it will fail are also high.

It will probably be necessary to make contact with the maintainers of the distribution you are using to see if there is an additional repository with newer packages.

**Reply by Josh65-2201** [reply]
~~Do you know what distributions have ben tested to work with easy effects~~
Ok so I saw the list of test distributions, I update apt to support Debien's testing channel (Debian's 13 package's) which gives me Easyeffects 7.1.4 and Pipewire 1.0.3
